packages/native-stack/src/types.tsx

```diff
@@ -452,6 +454,43 @@ export type NativeStackNavigationOptions = {
    * @platform ios
    */
   gestureResponseDistance?: ScreenProps['gestureResponseDistance'];
+  /**
+   * Changes the gesture for dismissing the screen to the previous one.
+   *
+   * Supported values:
+   * - "swipeRight": swipe right to dismiss the screen
+   * - "swipeLeft": swipe left to dismiss the screen
+   * - "swipeUp": swipe up to dismiss the screen
+   * - "swipeDown": swipe down to dismiss the screen
+   * - "verticalSwipe": swipe up or down to dismiss the screen
+   * - "horizontalSwipe": swipe left or right to dismiss the screen
+   * - "twoDimensionalSwipe": swipe in any direction to dismiss the screen
+   *
+   * Only supported on iOS and Android.
+   */
+  goBackGesture?: GoBackGesture;
+  /**
+   * Changes the animation for dismissing the screen when making a swipe gesture.
+   * You can choose one of the built-in transition presets, by using `ScreenTransition` from `react-native-reanimated`.
+   *
+   * Supported values:
+   * - ScreenTransition.SwipeRight
+   * - ScreenTransition.SwipeLeft
+   * - ScreenTransition.SwipeUp
+   * - ScreenTransition.SwipeDown
+   * - ScreenTransition.Horizontal
+   * - ScreenTransition.Vertical
+   * - ScreenTransition.TwoDimensional
+   * - ScreenTransition.SwipeRightFade
```

Comment:
Have you considered a string union type instead? So we don't need an additional import.

Reply:
As above, this could be a problem here aswell, since `ScreenTransition` object isn't exported from react-native-screens. Should we wait for the release that exports that object then?

Reply:
Missed that this was from reanimated. Then it seems fine.